src/main/java/seedu/address/model/module/LessonType.java

```diff
@@ -0,0 +1,17 @@
+package seedu.address.model.module;
+
+/**
+ * Lesson type of the Lesson.
+ */
+public class LessonType {
```

Comment:
I'm not sure if the type of lessons are limited and can be known but if so, could this class be turned into an enum instead?

Reply:
Good point. Unfortunately the API doesn’t provide any guarantees for LessonType (there might be new lesson types in the future). I’ll look into using enums for other classes (e.g. SemesterNo) in the next release.